core/src/main/java/io/confluent/connect/storage/StorageSinkConnectorConfig.java

```diff
@@ -92,6 +94,19 @@
   public static final int SCHEMA_CACHE_SIZE_DEFAULT = 1000;
   public static final String SCHEMA_CACHE_SIZE_DISPLAY = "Schema Cache Size";
 
+  public static final String ENHANCED_AVRO_SCHEMA_SUPPORT_CONFIG = "enhanced.avro.schema.support";
+  public static final boolean ENHANCED_AVRO_SCHEMA_SUPPORT_DEFAULT = false;
```

Comment:
is there a down side to not make this default behavior?

Reply:
Good point. I'm inclined to be conservative at this point, and enabled it by default in next regular version (`4.1.0`) instead of the bugfix version (`4.0.1`).

Reply:
👍

Reply:
Would enabling this by default break any existing configurations or behavior?

Reply:
It shouldn't I believe, given that it adds support for unions and enums, but we'll have the chance to test even more.

Reply:
if it's purely bugfix, put it in the earlier branch as long as it isn't risky. also, if that's the case, would we really need a config?